haiku-games/hexvexed/hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION=""HexVexed is a maddening addictive and easy to play puzzle game:
+Arrange hexagonal tiles into a grid so that the numbers and colours match
+where they touch. This game is a version DarkWyrm's BeVexed, but instead
+of squares, the tiles a hexagonal shaped. It's 1.5x the fun!
+
+HOMEPAGE="https://bitbucket.org/scottmc/hexvexed"
+COPYRIGHT="2009-2016 Scott McCreary"
+LICENSE="MIT"
+REVISION="1"
+commit="68437a22c788"
+SOURCE_URI="$HOMEPAGE/get/$commit.zip"
+CHECKSUM_SHA256="35e0904d91f15fefe801cdb7b9f9bdf4dc925de69f5b8818bec24cc4c22dc6d7"
+SOURCE_DIR="scottmc-hexvexed-$commit"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+
+PROVIDES="
+	hexvexed = $portVersion
+	app:HexVexed = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+	
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:g++
+	"
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects
+	make $jobArgs OBJ_DIR=objects bindcatalogs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	mkdir -p backgrounds
+	mkdir -p $dataDir/HexVexed
+
+	cp -r backgrounds $dataDir/HexVexed
+      
+	cp -a objects/HexVexed $appsDir
+        addAppDeskbarSymlink $appsDir/HexVexed
```

Comment:
Can you please fix the indentation in INSTALL?
Some lines have multiple spaces instead of tab(s).

BTW, you might also wish to use the install command.
It is a little bit better than using mkdir anc cp.
For examples on how to use use install, you can have a look in:
media-gfx/apng2gif/
media-gfx/gif2apng/
media-gfx/png2ico/
app-misc/hexcompare/
app-crypt/stegsnow/

Thanks!